Add e2e tests for the remote config updater (#1243)
DataDog/datadog-operator: docs/how-to-contribute.md

# Run E2E tests with K8S_VERSION, IMG, and IMAGE_PULL_PASSWORD environment variables (for pulling operator image from a private registry).
$ K8S_VERSION=1.25 IMG=669783387624.dkr.ecr.us-east-1.amazonaws.com/operator:PIPELINE_ID-COMMIT_HASH IMAGE_PULL_PASSWORD=$(aws-vault exec sso-agent-qa-read-only -- aws ecr get-login-password) aws-vault exec sso-agent-sandbox-account-admin -- make e2e-tests
```
> **NOTE:**  The remote configuration updater test requires the owner of the API Key to have the permission `Fleet Policies Write`. 
> To get the permission:

> >- Log in the ddev org

Reviewer: ```suggestion
>- Log into the ddev org.
```

> >- Go to the [roles page](https://dddev.datadoghq.com/organization-settings/roles)

Reviewer: ```suggestion
>- Go to the [roles page](https://dddev.datadoghq.com/organization-settings/roles).
```

> >- Search for `Fleet Policies Write`, Click on it

Reviewer: ```suggestion
>- Search for `Fleet Policies Write`, and click on the result.
```

> >- Click on `request role`

Reviewer: ```suggestion
>- Click on **request role**.
```

> >- It should be added after few minutes.

Reviewer: ```suggestion
>- The role should be added after a few minutes.
```
